srcpkgs/grub2-theme-preview/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'grub2-theme-preview'
+pkgname=grub2-theme-preview
+version=2.8.0
+revision=1
+archs="x86_64 x86_64-musl i386 i386-musl"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="grub mtools qemu xorriso"
+checkdepends="${depends} python3-parameterized"
+short_desc="Preview a full GRUB 2.x theme using KVM / QEMU"
+maintainer="Animesh Sahu <animeshsahu19@yahoo.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/hartwork/grub2-theme-preview"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=f96dda1dce58b3336ccae7697640490941bcfcce567352c7f4f5a8f4d2b193df
+
+case $XBPS_TARGET_MACHINE in
+	x86_64*) depends+=" grub-x86_64-efi"; checkdepends+=" grub-x86_64-efi" ;;
+	i686*) depends+=" grub-i386-efi"; checkdepends+=" grub-i386-efi" ;;
+	*) broken="not supported" ;;
```

Comment:
not necessary, this is already implied by archs=

Reply:
A valid grub-efi implementation is required for both runtime to work as well as tests to pass. I don't think there's an easy way to embed this in `depends=` at the metadata declaration itself. What do you think?